ConnectionsPanel.js

```diff
@@ -319,92 +355,18 @@ class ConnectionsPanel extends React.PureComponent {
-          />
-          ) : null
-        }
-        <ToolsButton
-          text={strings.sheets}
-          icon={iconData.get('sheet', this.props.themeStr)}
-          count={this.props.sheetsCount}
-          onPress={()=>{ this.props.setConnectionsMode("sheetsByRef"); }}
-        />
-        {this.props.topicsCount && this.props.topicsCount > 0 ? (
-          <ToolsButton
-            text={strings.topics} count={this.props.topicsCount}
-            icon={iconData.get('hashtag', this.props.themeStr)}
-            onPress={() => this.props.setConnectionsMode("topicsByRef")}
-          />) : null
-        }
-        <ToolsButton
-          text={strings.about}
-          icon={iconData.get('book', this.props.themeStr)}
-          onPress={()=>{ this.props.setConnectionsMode("about"); }}
-        />
-        {this.props.sheet ? null : <ToolsButton
-          text={strings.translations}
-          icon={iconData.get('layers', this.props.themeStr)}
-          count={this.props.versionsCount}
-          onPress={()=>{ this.props.setConnectionsMode("versions"); }}
-        /> }
-        <ToolsButton
-          text={strings.share}
-          icon={iconData.get('share', this.props.themeStr)}
-          onPress={() => this.props.shareCurrentSegment()}
-        />
-        {this.props.sheet ? null : <ToolsButton
-          text={strings.reportError}
-          icon={iconData.get('bubble', this.props.themeStr)}
-          onPress={this.props.reportError}
-        />}
-        <ToolsButton
-          text={strings.viewOnSite}
-          icon={iconData.get('externalLink', this.props.themeStr)}
-          onPress={this.props.viewOnSite}
-        />
-      </View>
-    );
-  }
-}
-
 const ToolsButton = ({ text, onPress, icon, count }) => {
   const { themeStr, interfaceLanguage } = useContext(GlobalStateContext);
   const theme = getTheme(themeStr);
   const textStyle = interfaceLanguage === "english" ? styles.enInt : styles.heInt;
   const flexDir = interfaceLanguage === "english" ? null : styles.rtlRow;
   const hasIcon = !!icon;
   icon = icon || iconData.get('sheet', themeStr);  // default to arbitrary icon that will be invisible if icon wasn't passed
-  const iconComp = (<View style={[styles.toolsButtonIcon, hasIcon ? null : styles.readerNavSectionMoreInvisible]}><Image source={icon} style={styles.menuButton} resizeMode={'contain'}></Image></View>);
+  const iconComp = (<View style={[styles.toolsButtonIcon, flexDir, hasIcon ? null : styles.readerNavSectionMoreInvisible]}><Image source={icon} style={styles.menuButton} resizeMode={'contain'}></Image></View>);
   const countComp = !!count || count === 0 ? <Text style={[styles.enInt, theme.secondaryText, styles.spacedText]}>{` (${count}) `}</Text> : null
   return (
     <SefariaPressable
-      extraStyles={[styles.searchFilterCat, styles.toolsButton, flexDir, theme.bordered]}
+      extraStyles={[styles.searchFilterCat, styles.toolsButton, flexDir, theme.bordered, {height: 36}]}
```

Comment:
create a new style in Styles.js for the height. By passing a literal object as a style it forces the component to rerender whenever the parent rerenders.